src/tutorial/styles.scss

```
@@ -28,3 +28,11 @@
 
   }
 }
+
+.tutorialsText {
+  padding: 0.5rem 0.5rem;
+}
+
+.tutorialsText:hover {
+  background: #e8e8e8;
```

```suggestion
  background: colors.$white-hover;
```

[Carbon color token mappings](https://carbon-elements.netlify.app/colors/examples/preview/)